docs/user/guides/migrate-rest-myst.md

```diff
@@ -27,8 +27,8 @@ and some others in reStructuredText, for whatever reason.
 Luckily, Sphinx supports reading both types of markup at the same time without problems.
 
 To start using MyST in your existing Sphinx project,
-first {ref}``install the `myst-parser` Python package <myst-parser:intro.md#installation>``
-and then {ref}`enable it on your configuration <myst-parser:intro.md#enable-myst-in-sphinx>`:
+first [install the `myst-parser` Python package](https://myst-parser.readthedocs.io/en/stable/intro.html#installation)
+and then [enable it on your configuration](https://myst-parser.readthedocs.io/en/stable/intro.html#enable-myst-in-sphinx):
```

Comment:
Why are we changing intersphinx references for regular Markdown links?

Reply:
See PR description: https://github.com/readthedocs/readthedocs.org/pull/10090#issue-1604951450

Reply:
Oh, this is the particular bug you found. :+1:

Reply:
Oh, this is the particular bug you found. :+1:

Reply:
Oh, this is the particular bug you found. :+1:

Reply:
It was a bit tedious to brute force through all the likely ideas of what combination myst-parser would somehow accept until I gave up and looked in objects.inv and didn't find what I wanted to reference :)

I think it's best if we just update this reference, we do still use Intersphinx for myst-parser, so we can keep the mapping.